docs/install/any-kubernetes-cluster.md

```diff
@@ -730,12 +730,24 @@ kubectl label namespace default eventing.knative.dev/injection=enabled
 
 {{< feature-state version="v0.2" state="alpha" >}}
 
-The following command installs the Github Source:
+The following command installs the single-tenant Github source:
```

Comment:
see suggestions above - also why do we need to duplicate this content so that it's maintained in two places? Can a link or some way of including the file not be used instead?

Reply:
I removed the section in the sample and kept the link.